.circleci/config.yml

```diff
@@ -70,6 +94,7 @@ jobs:
   version_validation:
     macos:
       xcode: "15.0.0"
+    resource_class: macos.x86.medium.gen2
```

Comment:
Does this resource still exist? I thought circle ci sunset it about 6 months ago